src/article.js

```diff
@@ -487,3 +487,35 @@ Article.prototype.handleResize_ = function() {
     this.sections[i].rerender();
   }
 };
+
+/**
+ * Private method
+ * Removes blank paragraphs components from direction start/end
+ * @param {object} `component` specifiy the start from component.
+ * @param {boolean} `end` detrimines if direction is end.
+ */
```

Comment:
We use @private decorator to indicate a private function. Also we usually use suffix _ for methods.